presto-spark-base/src/main/java/com/facebook/presto/spark/execution/NativeExecutionProcess.java

```diff
@@ -83,21 +78,15 @@
     private final ScheduledExecutorService errorRetryScheduledExecutor;
     private final RequestErrorTracker errorTracker;
     private final HttpClient httpClient;
-    private final NativeExecutionTask nativeExecutionTask;
 
     private Process process;
 
     public NativeExecutionProcess(
```

Comment:
Do you think if it's possible to add some tests to test multiple instances of NativeExecutionProcess been launched successfully in the same host?

Reply:
Based on offline discussion, will add test around the `NativeExecutionProcess ` in next PR.